kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -269,29 +296,39 @@ define([
             $tabsDiv.empty();
             var data = JSON.parse(dataString);
 
+            var ulCSS = {
```

Comment:
why not make this a class?

Reply:
I hate external class files. :-P

I think it's tougher to keep 'em in sync with the code, and cruft accumulates too easily when usage of the class is deleted but the actual class still hangs around in the stylesheet.

Nonetheless, I retooled to use a class, and added them into kbase-extension/static/kbase/css/kbaseNarrative.css

Lemme know if they should be in a different file.